### ISSUE TITLE
Pillar Tracker for Specific Pillar

### ISSUE THREAD

**Issue by 0x3639** [zenon-tools/pillar-tracker#1]
Would it be possible to add tracking for a specific pillar?  So for those who delegate to deeZNNutz.com, for example, they could subscribe to a specific channel, turn on notifications, and only get pillar messages for the specific pillar (rather than all pillars).  

Ideally the tracking would be a list in the config file.  That way someone could track, 1, a few, or all pillars.  We should leave all other notifications unchanged, such as new pillars, removed pillars, and when rewards are ready.  

Happy to pay for this feature too!

**Comment by vilkris4** [comment]
This should be possible yes. It might make sense that the Pillar owner would host the channel and just provide the channel's ID to Zenon Tools and then add the Pillar Tracker bot to the channel as an admin. The channel would then receive updates about the Pillar in question, like if the pillar stops or changes its rewards.

**Comment by 0x3639** [comment]
Yes, this is exactly what I was thinking!  I think this change would be super useful to specifically track one pillar - for both the operator and delegator.

**Comment by 0x3639** [comment]
I thought of a way to help you get some donations for this work.  I will offer to host this script and setup the bots for all Pillars who want to run it so long as they donate 20 ZNN to you.  I have a lot of infrastructure so it does not cost me anything to run several versions of the script.  I can then setup bots for each pillars, map them to the script, and run it for pillars who want to use it.  And you can then gets some nice donations.

**Comment by vilkris4** [comment]
Thank you for the offer and idea. The way that it would work wouldn't require hosting multiple bots or scripts though because one bot and script (the current ones) would be responsible for sending all the messages to the different channels - this just needs a few additions to the current code.

This feature could be used as a proof-of-concept for offering Pillars services for a one-time fee or a monthly subscription that the Pillar owner could control on zenon.tools.

Happy to try this idea out with your Pillar.

**Comment by 0x3639** [comment]
Cool.  If this works I have another idea we can expand on for services you could charge for.  As you know these nodes seem to crash when memory usage grows.  Pillars seem to simply restart their servers every 24 hours to prevent this.  

I'm testing a service called Monit.  If can be used to monitor running services.  You can monitor memory usage and then report it and even restart the service when it reaches certain limits.  Today, the services goes to 100% mem usage, crashes the daemon, and restarts.  Sometimes it gets hung in that process.     Proactively restarting it could be good.  I'm testing that now.

Monit has a push notification system.  It can send emails and push messages to the phone

https://mmonit.com/wiki/MMonit/PushoverNotification

Pillars and node operators might want this push notification service from you.  To make it easy.  Operators would install monit, and send notifications to you for distribution.  I'm testing now and will let you know hit it works.

**Comment by vilkris4** [comment]
That's an interesting idea, I'll give that a look.
